plugin/src/it/copy-provided-pages/pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.bonitasoft.test</groupId>
+  <artifactId>my-project-parent</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <packaging>pom</packaging>
+  <modules>
+    <module>app</module>
+  </modules>
+  <properties>
+    <maven.compiler.release>11</maven.compiler.release>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <bonita-project-maven-plugin.version>@project.version@</bonita-project-maven-plugin.version>
+    <bonita.runtime.version>8.0.0</bonita.runtime.version>
```

Comment:
As we will publish user pages artifacts from 9.0.0, what should we do with this test? Using 9.0.0-SNAPSHOT ?

Reply:
I recommend testing it in bonita-project in the meanwhile.

Reply:
I've done it partially.
But it is strange to have a test on code that is not in the same repository.

Reply:
I suggest waiting for 9.0.0 release and adding the test at this level later on.
The other solution is to artificially install fake zip archives with the expected coordinate in a first invocation in the `invoker.properties` file, like this:

```properties
invoker.name.1 = Install fake page
# page.zip is an archive file at the root of the test project
invoker.goals.1 = install:install-file -Dfile=page.zip -Dartifact=org.bonitasoft.web.page:page-user-task-list:8.0.0:zip
invoker.buildResult.1 = success
invoker.name.2 = Test copying provided pages
invoker.goals.2 = ${project.groupId}:${project.artifactId}:${project.version}:copy-provided-pages
invoker.buildResult.2 = success

```

Reply:
Yes it can be a temporary solution waiting for the 9.0.0 release